dev-python/autobahn/autobahn-0.16.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_4 )
```

Comment:
In general you shouldn't remove py3.3, as it could break the depgraph for other people.